Readme.md

```diff
@@ -51,7 +50,7 @@ window.onresize.force();
   The debounced function returned has a property 'clear' that is a 
   function that will clear any scheduled future executions of your function.
 
-  The debounced function returned has a property 'force' that is a 
+  The debounced function returned has a property 'flush' that is a 
   function that will immediately execute the function if execution is scheduled,
```

Comment:
please replace "if" with "if and only if" - I had misunderstood the first time I read it.